### PR TITLE
ZA2 - Temwa Nyirenda - Week #6 Coursework (JavaScript Core I - Lesson 1)

### PR DESCRIPTION
**Pull request by TemwaNyirenda**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Temwa Nyirenda
- Your City: South Africa
- Your Slack Name: Temwa Nyirenda

# Homework Details

- Module: JavaScript Core I - Lesson 1
- Week: 6

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

**Review by mfukechristian — COMMENTED** [review]
Very nice  , very very nice

**Review by chandredewet — COMMENTED** [review]
Hi Temwa,

Wow, I see your maths have helped you in sales tax with * 1.2.

File: exercises/L-functions-nested/exercise.js [hunk 1]
```diff
@@ -3,3 +3,23 @@ var mentor2 = "Irina";
...
+}
+
+function shoutName(name) {
+	return (name.toUpperCase());
+}
+
+greeting = "Finally you're here! Come on in, ";
+
+function createAShoutyMessageWithName(message, name) {
+	return (createAShoutyMessage(message) + shoutName(name));
```
Comment (chandredewet) [inline exercises/L-functions-nested/exercise.js:18]: Hi Temwa,

Great work, just a suggestion, u could have used the same function, e.g shoutName  perhaps just named differently to capitalise the shouty message.

**Review by chandredewet — COMMENTED** [review]
Hi Temwa,

Apologies for the limited comments, but well done on interesting greetings throughout and doing most of the extra exercises, wow, your JS is going to be strong as a result.

I have only made one change - the use of the function in shouty greeting. and would like to commend u on knowing how to work out mathematically the price including 20%

**Review by diegoharera — COMMENTED** [review]
Hi @TemwaNyirenda, These exercises were good to understand the basic Javascript language. declaring functions, variables, parameters, arguments, and naming conventions, were the goals for week1, and you have shown great skills in the basic javascript language. Well Done!

**Pull request closed without merging**